pydle/features/tls.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, *args, tls_client_cert=None, tls_client_cert_key=None, tls_cl
                 port = DEFAULT_TLS_PORT
             else:
                 port = rfc1459.protocol.DEFAULT_PORT
-        return await super().connect(hostname, port, tls=tls, **kwargs)
+        return super().connect(hostname, port, tls=tls, **kwargs)
```

Comment:
In this instance, the super call is still asynchronous, I don't see why we would want to return a coroutine object rather than awaiting the super